deploy/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: aws-account-shredder-template
+
+parameters:
+  - name: IMAGE
+    required: true
+    value: quay.io/agautam/aws-account-shredder
+  - name: IMAGE_TAG
+    value: latest
+
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: aws-account-shredder
+      labels:
+        name: aws-account-shredder
+    spec:
```

Comment:
Add update strategy here. for example https://github.com/app-sre/github-mirror/blob/master/openshift/github-mirror.yaml#L24-L28

Reply:
added an update strategy